api/v1alpha1/contentlibraryitem_types.go

```diff
@@ -131,6 +148,12 @@ type ContentLibraryItemStatus struct {
 	// +optional
 	FileInfo []FileInfo `json:"fileInfo,omitempty"`
 
+	// +optional
+
+	// StorageInfo represents the storage backing information of the files
+	// belonging to the content library item in vCenter.
+	StorageInfo []StorageInfo `json:"storageInfo,omitempty"`
```

Comment:
Since the storage info is per file, instead of having a separate list, it can merge with the fileInfo list, any specific reason for maintaining 2 separate list?

Reply:
I placed it in a different field because the information is retrieved from a separate API. From this CRD's perspective, it makes sense to move it under the existing FileInfo. I've updated the PR to address this.